.github/workflows/pip.yml

```diff
@@ -0,0 +1,52 @@
+name: Pip
```

Comment:
Should we regularly (every day, every few days) trigger this? In general, these repositories won't be update that often, and we'd like to know if a recent version of pybind11 breaks it?

Reply:
I'd go for weekly, I think.

Reply:
The weekly job would need to run off master, though. Easier here than in the other two that pull from PyPI.

Reply:
Ah, yes, this works with the release. Well, maybe we can just make this part of the release protocol, otherwise.

Reply:
Btw, I'm more worried about CI dependencies breaking (as they have often been) than about changes to pybind11 breaking this project.

Reply:
It's not hard for this one, just add a step that does a git pull on the submodule if this was triggered by the weekly trigger.

Reply:
If you use dependabot + fixed actions, it should be pretty stable, I'd think. But I think the piecemeal design of GHA/Azure is much more future-proof than older CI's that upgrade out from under you.

Reply:
That's also true, yes. I'm talking about my Travis traumas, every time HomeBrew or pyenv or ... was updated. So as far as I'm concerned, we can drop this, then, for now. If in the future, things turn out to still break, we can always fix that?